elections/steering/2023/candidate-pacoxu.md

```diff
@@ -0,0 +1,47 @@
+-------------------------------------------------------------
+name: Paco Xu 徐俊杰
+ID: pacoxu
+info:
+
+- employer: DaoCloud
+- slack: pacoxu
+
+-------------------------------------------------------------
+
+## SIGS
+
+- SIG-ClusterLifecyle: kubeadm maintainer
```

Comment:
ClusterLifecyle should be ClusterLifecycle

Reply:
Updated.